examples/cosmos/ics20/ics20.qnt

```diff
@@ -255,6 +255,19 @@ module ics20Test {
     channelCounterparties.get(sourceChain).get(packet.sourceChannel) == packet.destChannel
   }
 
+  /* Helper operators to manipulate bank state */
+  /// Sets the balance of `account` in `chain` to `amount` of `denom`.
+  action SetBalanceIn(chain, account, denom, amount) = {
```

Comment:
so far, all our actions are lower camel case, this is upper camel case.
shall we use the same spelling on all of them?

Reply:
For some reason, I thought we were using upper camel case for actions, but we are not. Will fix.